src/select/select.ts

```diff
@@ -259,6 +259,7 @@ export class SelectComponent implements OnInit, ControlValueAccessor {
   @Input() public textField:string = 'text';
   @Input() public childrenField:string = 'children';
   @Input() public multiple:boolean = false;
+  @Input() public formControl = new FormControl(); 
```

Comment:
This is already allowed because the class implements ControlValueAccessor and controls the current value via an internal implementation.

With your implementation the formControl's value would always be exactly the internal <select>'s value instead of the actual SelectItem[] selection, which is not the way this component was designed I think.

(Also, this repo has been dead for months unfortunately)